sormas-api/src/main/java/de/symeda/sormas/api/event/EventExportDto.java

```diff
@@ -486,6 +489,15 @@ public void setContactCountSourceInEvent(long contactCountSourceInEvent) {
 		this.contactCountSourceInEvent = contactCountSourceInEvent;
 	}
 
+	@Order(42)
```

Comment:
I'd suggest to add the event management status next to the investigation status or event status, not to the very end